### DISCUSSION TITLE
[confidence] Option not to raise an MR if confidence is low

### DISCUSSION THREAD

**Discussion by slonim-greg** [renovatebot/renovate#23944] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.27.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

A configurable ability not to raise an upgrade MR if release version confidence is low [for supported libraries].

Apologies if this was discussed before.

### What you tried so far.

Nothing.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
This is available using the `matchConfidence` matcher in `packageRules`, however it requires access to Mend's API key which is restricted to customers only.

https://docs.renovatebot.com/configuration-options/#matchconfidence

**Reply by slonim-greg** [reply]
@rarkins - is using Mend's API key (with a paid support) compatible with a self-hosting Gitlab setup? Additionally, how can I inquire of the cost/conditions of Renovate's professional services?

**Reply by rarkins** [reply]
Yes it is. Please see https://www.mend.io/contact-us/
